src/main/java/com/saralein/server/response/DirectoryResponse.java

```diff
@@ -0,0 +1,33 @@
+package com.saralein.server.response;
+
+public class DirectoryResponse implements Response {
+    private final String contentType = "text/html";
+
+    public byte[] createResponse() {
+        StringBuilder response = new StringBuilder();
+        response.append(createHeader());
+        response.append(createBody());
+
+        return response.toString().getBytes();
+    }
+
+    private String createHeader() {
+        return new Header("200 OK", contentType).getContent();
+    }
+
+    private String createBody() {
+        FileHelper fileHelper = new FileHelper("public");
```

Comment:
Something about the file helper being initialized on every call to `createBody` seems unnecessary to me, and then you're tied to this specific implementation of FileHelper. It's kind of rigid.